instrumentation/armeria-1.3/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/armeria/v1_3/ArmeriaSingletons.java

```diff
@@ -19,7 +22,17 @@ public final class ArmeriaSingletons {
   public static final Function<? super HttpService, ? extends HttpService> SERVER_DECORATOR;
 
   static {
-    ArmeriaTelemetry telemetry = ArmeriaTelemetry.create(GlobalOpenTelemetry.get());
+    ArmeriaTelemetry telemetry =
+        ArmeriaTelemetry.builder(GlobalOpenTelemetry.get())
+            .setCapturedClientRequestHeaders(CommonConfig.get().getClientRequestHeaders())
+            .setCapturedClientResponseHeaders(CommonConfig.get().getClientResponseHeaders())
+            .setCapturedServerRequestHeaders(CommonConfig.get().getServerRequestHeaders())
+            .setCapturedServerResponseHeaders(CommonConfig.get().getServerResponseHeaders())
+            .addClientAttributeExtractor(
```

Comment:
This client/server method split is a bit awkward; I'm starting to think that there should be separate library instrumentation entry points for client/server, consumer/producer etc.

Reply:
this makes sense to me, I'd guess that most usage locations are dealing with client or server and not both(?)